README.md

```diff
@@ -73,18 +75,36 @@ e.g.: https://discord.com/api/invite/discord-townhall - the guild's features are
 
 * `WELCOME_SCREEN_ENABLED` - Has welcome screen enabled, a modal shown to new joiners that features different channels and a short description of the guild.
 
+* `TICKETED_EVENTS_ENABLED`— Ability to view and manage ticketed events.
+
+* `MONETIZATION_ENABLED` — Has ticketed stages enabled.
+
+* `MORE_STICKERS` — Sets the sticker slots to 60. Not used in server boosting.
+
+* `HUB` — Not released
```

Comment:
Could use a more descriptive doc, whether the feature is currently available or not